content/authors/neema-muganga/index.md

```diff
@@ -0,0 +1 @@
+Neema Muganga is a Tech enthusiast who enjoys creating web applications using ReactJS and Django tools. Networking with peers, exploring and learning new tech skills particularly interests her.
```

Comment:
Please check how other contributors have created their profile - index.md. You need to use the same style. You can find this information in the `content/authors/` folder.